doc/source/user_guide/dsintro.rst

```diff
@@ -663,6 +672,31 @@ row-wise. For example:
 
    df - df.iloc[0]
 
+In the special case of working with time series data, if the DataFrame index
```

Comment:
This is being added to the documentation?

Reply:
Removed--I was attempt to resolve a merge error (and inadvertently added this).